torch_uncertainty/routines/classification.py

```diff
@@ -453,6 +493,11 @@ def add_model_specific_args(
         parent_parser.add_argument(
             "--logits", dest="use_logits", action="store_true"
         )
+        parent_parser.add_argument(
```

Comment:
Actually, I found it quite fast so the main issue with the GL is that we need the models to have a feats_forward maybe

Reply:
Is that something we could check beforehand? To make sure nobody asks for the grouping_loss without the `feats_foward()` method in their model?

Reply:
We can check it with hasattr at the beginning of the routine perhaps?